docs/faq/security.md

```diff
@@ -7,12 +7,16 @@ title: Security FAQ
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/faq/security"/>
 </head>
 
-**Is there a Hardening Guide?**
+### Is there a Hardening Guide?
 
 The Hardening Guide is now located in the main [Security](../reference-guides/rancher-security/rancher-security.md) section.
 
-<br/>
-
-**What are the results of Rancher's Kubernetes cluster when it is CIS benchmarked?**
+### What are the results of Rancher's Kubernetes cluster when it is CIS benchmarked?
```

Comment:
```suggestion
### Where can I access results after running a CIS benchmark on a hardened Rancher Kubernetes cluster?
```

Reply:
Just a suggestion but perhaps something like above to rephrase the header?

Reply:
Thanks for giving it a try. I am having a hard time with this heading title and think I'll shelve it for a later PR. Maybe the point it's trying to get across is that we have self-assessment guides in the Security section of the docs? @btat could you weigh in?

Reply:
I committed @btat's suggestion to the branch